core/server/automations.go

```diff
@@ -57,25 +56,3 @@ func (cs *coreServer) GetKustomization(ctx context.Context, msg *pb.GetKustomiza
 
 	return &pb.GetKustomizationResponse{Kustomization: res}, nil
 }
-
-func (cs *coreServer) ListHelmReleases(ctx context.Context, msg *pb.ListHelmReleasesRequest) (*pb.ListHelmReleasesResponse, error) {
```

Comment:
Moving this to a new file makes the naming inconsistent. If we are really going for that level of separation, we also need to rename `automations` to `kustomizations`, since that is all it contains now.

Reply:
I would like to split them up, later on much easier to find and change without worrying about a conflict between PRs. I'll rename the file to `helm_release` and `automations` to `kustomization`